### DISCUSSION TITLE
Hide "Action Button" on Details Page if  Field "Yes/No" is YES?

### DISCUSSION THREAD

**Discussion by MickGraham23** [cortexrd/Knack-Toolkit-Library#340] [Q&A]
Is this available on Details Pages and if so, can some help me with the syntax?

Thanks
Michael :)

**Comment by cortexrd** [comment]
Hi Michael,

I'd go for this:

`_cfv=[YesNoFieldHeader, is, Yes,,,,h], [ktlTarget, Action Button]`

_cfv is supported by details views.

Note that I haven't test this line above, yet.

Give it a try and let us know if it works.
Norm
